apps/hash-api/src/graphql/resolvers/index.ts

```diff
@@ -188,9 +184,10 @@ export const resolvers = {
     results: linkedAggregationResults,
   },
 
-  // New knowledge field resolvers
   Page: {
+    // @ts-expect-error –– the type requires 'blockChildEntity' inside the return, but we deal with it in a field resolver
     contents: pageContents,
+    // @ts-expect-error –– the type requires 'contents' to be returned here, but we deal with it in a field resolver
     parentPage: parentPageResolver,
   },
```

Comment:
This would make the type definition of this object more complicated, but you could try and make use of the existing `UnresolvedBlockGQL` and `UnresolvedPageGql` types to get rid of these TS errors

Reply:
I'm going to leave this for now since I don't think these resolvers need to exist anyway (the frontend could figure it out from a subgraph) and don't want to sink more time into htem